event/register_oracle_event.go

```diff
@@ -89,8 +81,25 @@ func (e RegisterOracleEvent) EventHandler(event ctypes.ResultEvent) error {
 	return nil
 }
 
-// verifyReportAndGetVoteOption validates the RemoteReport and returns the voting result according to the verification result.
-func verifyReportAndGetVoteOption(oracleRegistration *types.OracleRegistration, e RegisterOracleEvent) types.VoteOption {
+// verifyAndGetVoteOption performs a verification to determine a vote.
+// - Verify that trustedBlockInfo registered in OracleRegistration is valid
+// - Verify that the RemoteReport is valid
+func verifyAndGetVoteOption(oracleRegistration *types.OracleRegistration, e RegisterOracleEvent) types.VoteOption {
+	block, err := e.reactor.QueryClient().GetLightBlock(oracleRegistration.TrustedBlockHeight)
+	if err != nil {
+		log.Warnf("failed to get lightBlock. height(%v), error(%v)", oracleRegistration.TrustedBlockHeight, err)
+		return types.VOTE_OPTION_NO
```

Comment:
If an error occurs when getting light block, I think we should return error and not vote for no.
In this case, something's wrong in oracle itself and this oracle cannot judge the validity of oracle registration.

Reply:
I first developed it as @H4NLee said.
However, if the height of the parameter is higher than the current block, an error occurs in this process.

Umm, I think it would be better to vote no for certain errors and not vote for the rest.

Reply:
Ah got it. Then, maybe [`ErrHeightTooHigh`](https://github.com/tendermint/tendermint/blob/c69ab6884837ad7a9e6ebadc2598d702bd14f5d5/light/provider/errors.go#L11)?

Reply:
Right!

Reply:
I fixed it. (https://github.com/medibloc/panacea-doracle/pull/79/commits/f082261652308262a129849d39f7976d72635afa)